### PR TITLE
feat: create new b2b actor from integration event

### PR DESCRIPTION
**Pull request by MadsDue**
## Description 
Create new b2b actors when market participant raise 'ActorActivated' integration event. 
- [x] new integration event command mapper for event
- [x] raise CreateActorCommand

Next PR:
- https://github.com/Energinet-DataHub/opengeh-edi/pull/583

## References
https://app.zenhub.com/workspaces/mosaic-60a6105157304f00119be86e/issues/gh/energinet-datahub/team-phoenix/86

### CODE REVIEW

File: source/IntegrationTests/Infrastructure/IntegrationEvents/WhenAnActorActivatedIsAvailableTests.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    private readonly IDatabaseConnectionFactory _connectionFactory;
+
+    public WhenAnActorActivatedIsAvailableTests(DatabaseFixture databaseFixture)
+        : base(databaseFixture)
+    {
+        _connectionFactory = GetService<IDatabaseConnectionFactory>();
+    }
+
+    [Fact]
+    public async Task New_actor_event_was_received()
```
Comment (lasrinnil) [inline source/IntegrationTests/Infrastructure/IntegrationEvents/WhenAnActorActivatedIsAvailableTests.cs:41]:
```suggestion
    public async Task WhenNewActorEvent_IsReceived_ActorIsStored()
```

Comment (lasrinnil) [reply source/IntegrationTests/Infrastructure/IntegrationEvents/WhenAnActorActivatedIsAvailableTests.cs:41]: The existing test method name does not describe the expected behavior, only the action.

File: source/RequestResponse/source/RequestResponse/IntegrationEvents/ActorActivated.proto [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto3";
+import "google/protobuf/timestamp.proto";
+
+option csharp_namespace = "Energinet.DataHub.MarketParticipant.Infrastructure.Model.Contracts";
```
Comment (lasrinnil) [inline source/RequestResponse/source/RequestResponse/IntegrationEvents/ActorActivated.proto:19]: Is there a  particular reason we are not using the contract through the Nuget package from Energinet.DataHub.MarketParticipant.Integration.Model?

File: source/RequestResponse/source/RequestResponse/RequestResponse.csproj [hunk 3]
```diff
@@ -77,6 +77,9 @@ limitations under the License.
         <CompileOutputs>True</CompileOutputs>
         <Generator>MSBuild:Compile</Generator>
       </Protobuf>
+      <Protobuf Update="IntegrationEvents\AggregatedTimeSeriesRequest.proto">
```
Comment (lasrinnil) [inline source/RequestResponse/source/RequestResponse/RequestResponse.csproj:80]: why is this here?

File: source/IntegrationTests/Infrastructure/IntegrationEvents/WhenAnActorActivatedIsAvailableTests.cs (hunk 1 above)
Comment (MadsDue) [reply source/IntegrationTests/Infrastructure/IntegrationEvents/WhenAnActorActivatedIsAvailableTests.cs:41]: It just a new name convention for this repo, normally the test method names describes the behavior in this repo :)

File: source/RequestResponse/source/RequestResponse/IntegrationEvents/ActorActivated.proto (hunk 2 above)
Comment (MadsDue) [reply source/RequestResponse/source/RequestResponse/IntegrationEvents/ActorActivated.proto:19]: Because it is not in the Nuget package. I got it comfired by Alex as well.  

[I går 13:18] Alexei Mihalchuk
Der er ikke nuget-pakker eller andet, hvis det er hvad du tænker.

Comment (lasrinnil) [inline source/RequestResponse/source/RequestResponse/IntegrationEvents/ActorActivated.proto:19]: perhaps this should be placed into a MarketParticipant namespace?

**Review by lasrinnil — APPROVED** [review]

**Review by lasrinnil — APPROVED** [review]
